### ISSUE TITLE
NTR: Thymine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8801]
New term request Thymine high in urine (segal_030512182243): Thyminuria  UMLS: 2749270

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000128
- label: Elevated urinary thymidine level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: 
- def: The amount of thymidine in the urine, normalized for urine concentration, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8801
- synonyms:

**Comment by MickeySegal** [comment]
You created a finding for thymidine but what was requested was a finding for thymine.  But it turns out that thymidine is deficient in some diseases, so this finding is good.  But we also need a finding for thymine:
 https://www.sciencedirect.com/science/article/pii/S0925443910000724
"Urine and plasma samples of the patients showed strongly elevated levels of dihydrouracil and dihydrothymine and moderately elevated levels of uracil and thymine".

**Comment by pnrobinson** [comment]
reopening

**Comment by pnrobinson** [comment]
@MickeySegal 
thanks for pointing out the mixup.

Adding new term: 
- ID: HP:6000331
- label: Elevated urinary thymine level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: 
- def: The amount of thymine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8801
- synonyms:
